rollup.config.js

```diff
@@ -0,0 +1,49 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import cssbundle from 'rollup-plugin-css-bundle';
+import copy from 'rollup-plugin-copy';
+import { terser } from 'rollup-plugin-terser';
+import postcss from 'postcss';
+import postcssImport from 'postcss-import';
+
+const isProduction = !process.env.ROLLUP_WATCH;
+
+export default {
+  input: 'js/main.js',
+  output: {
+    file: 'public/bundle.js',
+    name: 'WoIstMarkt',
```

Comment:
What is this `name` used for? If it is used for being displayed as the project name please stick to `Wo ist Markt?`. If it is used as the identifier for an internal project artifact then it's fine.

Reply:
This name can be removed. Done with ce73057d9188d2a0737dbadd9d3c0d3469a53880